lms/djangoapps/verify_student/migrations/0012_populate_verification_deadlines.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
```

Comment:
FYI: @feanil this is a data migration to copy data from the course modes table to a new table.  I've tested this forwards and backwards in devstack.  This will read ~650 rows from MySQL in one query and create that many rows in the new table.